lib/screenshot.rb

```diff
@@ -46,7 +46,7 @@ def self.rectangle(rectangles)
     height = bottom_right_y - top_left_y
 
     # The new rectangle is constructed with all the co-ordinates calculated above
-    [top_left_x, top_left_y, width, height]
+    [top_left_x - padding, top_left_y - padding, width + padding * 2, height + padding * 2]
```

Comment:
Is there any specific reason why width and height is increased twice of padding?